src/main/java/net/mcreator/ui/minecraft/boundingboxes/JBoundingBoxList.java

```diff
@@ -18,33 +18,55 @@
 
 package net.mcreator.ui.minecraft.boundingboxes;
 
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
 import net.mcreator.element.types.interfaces.IBlockWithBoundingBox;
+import net.mcreator.io.FileIO;
 import net.mcreator.ui.MCreator;
 import net.mcreator.ui.component.util.PanelUtils;
 import net.mcreator.ui.help.IHelpContext;
 import net.mcreator.ui.init.L10N;
 import net.mcreator.ui.minecraft.JEntriesList;
+import net.mcreator.workspace.resources.Model;
 
 import javax.swing.*;
 import java.awt.*;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Objects;
-import java.util.stream.Collectors;
+import java.util.function.Supplier;
 
 public class JBoundingBoxList extends JEntriesList {
 	private final List<JBoundingBoxEntry> boundingBoxList = new ArrayList<>();
 	private final JPanel entries = new JPanel(new GridLayout(0, 1, 5, 5));
 
+	private final Supplier<Model> modelProvider;
+	private final JButton genFromModel = new JButton(L10N.t("elementgui.common.gen_from_block_model")) {

```

Comment:
This could be an object (eg. `MTechnicalButton`), with the function `L10N.functionalButton` as it seems this will be used on more places in the future

Reply:
✔️ Fixed